Unset function so spy from PATH can run (#48)
codehearts/shpy: README.md

touch my-new-file # outputs "call me twice, shame on me" to stderr, returns false
```

When developing tests for complex functions with long chained calls, source all of them and use spy with -u flag. The flag will unset declared function, so complex functions can be tested with a mix of spies and original functions. 

```sh
source my_script.sh # contains complex_function that calls file_check and directory_check

createSpy -o 'spy test' -u file_check

> complex_function # complex_function calls file_check, so it will output "spy test". However, without -u createSpy gives a warning that file_check is already declared and complex_function executes declared one instead of spy

Reviewer: Would this be easier to read on multiple lines?

```suggestion
complex_function # complex_function calls file_check, which is mocked to output "spy test"
                 # Without -u, createSpy would warn that file_check is already declared and wouldn't be mocked
```
